### PR TITLE
Remove support for Teams and add support for Organizations and Networks

### PR DESCRIPTION
**Pull request by lswainemoore**

### CODE REVIEW

File: quantaq/endpoints/devices/__init__.py [hunk 1]
```diff
@@ -16,6 +16,8 @@ def __init__(self, client) -> None:
     def list(self, **kwargs) -> list:
         """Return all (available) devices.
 
+        :param int org_id: Filter to devices owned by given org.
+        :param int network_id: Filter to devices owned by given network.
```
Comment (lswainemoore) [inline quantaq/endpoints/devices/__init__.py:20]: Note that this doesn't actually involve any code, just `kwargs`. Added it to address stuff like: https://github.com/quant-aq/py-quantaq/issues/14.

File: pyproject.toml [hunk 2]
```diff
@@ -5,7 +5,7 @@ description = "API wrapper and utils for QuantAQ, Inc."
 authors = ["David H Hagan <david.hagan@quant-aq.com>"]
 license = "Apache License 2.0"
 readme = "README.md"
-homepage = ""
+homepage = "https://quant-aq.github.io/py-quantaq"
```
Comment (lswainemoore) [inline pyproject.toml:8]: Had to put something here to get tests to run--is this correct?

**Review by jaredbriskman — APPROVED** [review]
Approved, although it would be good to regenerate the sphinx docs!

File: docs/generated/quantaq.endpoints.devices.Devices.get.rst [hunk 3]
```diff
@@ -1,4 +1,4 @@
-quantaq.endpoints.devices.Devices.get
+﻿quantaq.endpoints.devices.Devices.get
```
Comment (lswainemoore) [inline docs/generated/quantaq.endpoints.devices.Devices.get.rst:1]: i don't know what these changes are about, but they're annoying!

File: docs/api.rst [hunk 4]
```diff
@@ -39,13 +39,22 @@ the Usage section for instructions on how to use them.
     endpoints.users.Users.update
 
 
-.. rubric:: Teams Endpoints
+.. rubric:: Orgs Endpoints
```
Comment (dhhagan) [inline docs/api.rst]: We should never refer to them as "Orgs" publicly, but always "Organizations"

File: docs/api.rst [hunk 5]
```diff
@@ -39,13 +39,22 @@ the Usage section for instructions on how to use them.
...
 
 .. autosummary::
     :toctree: generated/
 
-    endpoints.teams.Teams.list
-    endpoints.teams.Teams.get
+    endpoints.orgs.Orgs.list
```
Comment (dhhagan) [inline docs/api.rst]: Please change to Organizations

File: docs/generated/quantaq.QuantAQAPIClient.rst [hunk 6]
```diff
@@ -35,7 +35,8 @@ quantaq.QuantAQAPIClient
       ~QuantAQAPIClient.headers
       ~QuantAQAPIClient.logs
       ~QuantAQAPIClient.models
-      ~QuantAQAPIClient.teams
+      ~QuantAQAPIClient.networks
+      ~QuantAQAPIClient.orgs
```
Comment (dhhagan) [inline docs/generated/quantaq.QuantAQAPIClient.rst]: Please change to Organizations

File: docs/generated/quantaq.client.APIClient.rst [hunk 7]
```diff
@@ -35,7 +35,8 @@ quantaq.client.APIClient
       ~APIClient.headers
       ~APIClient.logs
       ~APIClient.models
-      ~APIClient.teams
+      ~APIClient.networks
+      ~APIClient.orgs
```
Comment (dhhagan) [inline docs/generated/quantaq.client.APIClient.rst]: Same

File: docs/generated/quantaq.client.DevelopmentAPIClient.rst [hunk 8]
```diff
@@ -35,7 +35,8 @@ quantaq.client.DevelopmentAPIClient
       ~DevelopmentAPIClient.headers
       ~DevelopmentAPIClient.logs
       ~DevelopmentAPIClient.models
-      ~DevelopmentAPIClient.teams
+      ~DevelopmentAPIClient.networks
+      ~DevelopmentAPIClient.orgs
```
Comment (dhhagan) [inline docs/generated/quantaq.client.DevelopmentAPIClient.rst]: Same as above

File: docs/generated/quantaq.client.ProductionAPIClient.rst [hunk 9]
```diff
@@ -35,7 +35,8 @@ quantaq.client.ProductionAPIClient
       ~ProductionAPIClient.headers
       ~ProductionAPIClient.logs
       ~ProductionAPIClient.models
-      ~ProductionAPIClient.teams
+      ~ProductionAPIClient.networks
+      ~ProductionAPIClient.orgs
```
Comment (dhhagan) [inline docs/generated/quantaq.client.ProductionAPIClient.rst]: Same as above

File: docs/generated/quantaq.client.StagingAPIClient.rst [hunk 10]
```diff
@@ -35,7 +35,8 @@ quantaq.client.StagingAPIClient
       ~StagingAPIClient.headers
       ~StagingAPIClient.logs
       ~StagingAPIClient.models
-      ~StagingAPIClient.teams
+      ~StagingAPIClient.networks
+      ~StagingAPIClient.orgs
```
Comment (dhhagan) [inline docs/generated/quantaq.client.StagingAPIClient.rst]: Same as above

File: docs/usage.rst [hunk 11]
```diff
@@ -72,29 +72,55 @@ obtain your user account information, you can use the `whoami` method:
...
+^^^^^^^^^^^^^^^^^^^^^^
 
-You can retrieve a list of all the teams you belong to:
+You can retrieve a list of all the organizations visible to you:
 
 .. code-block:: python
 
-    >>> teams = client.teams.list()
-    >>> print (teams)
+    >>> orgs = client.orgs.list()
```
Comment (dhhagan) [inline docs/usage.rst]: Fix once "Orgs" is swapped to "Organizations"

File: pyproject.toml (hunk 2 above)
Comment (dhhagan) [reply pyproject.toml:8]: huh. I wonder when that requirement changed!

File: quantaq/client.py [hunk 12]
```diff
@@ -45,7 +45,8 @@ def __init__(
 
         self._users = None
         self._devices = None
-        self._teams = None
+        self._orgs = None
```
Comment (dhhagan) [inline quantaq/client.py]: This is fine if only internal, though could consider switching to `_organizations` to stay self-consistent

File: quantaq/client.py [hunk 13]
```diff
@@ -216,12 +217,20 @@ def users(self):
         return self._users
 
     @property
-    def teams(self):
+    def orgs(self):
```
Comment (dhhagan) [inline quantaq/client.py]: Let's switch to Organizations as that's the public-facing name of the feature/tool

File: quantaq/endpoints/networks/__init__.py [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+        :param int org_id: The parent org id
+        :param int network_id: The network id
+
+        :returns: Network information
+        :rtype: dict
+        """
+        org_id = kwargs.pop("org_id")
+        network_id = kwargs.pop("network_id")
+        
+        return self.client.requests(f"orgs/{org_id}/networks/{network_id}", **kwargs)
```
Comment (dhhagan) [inline quantaq/endpoints/networks/__init__.py]: Is that what we stuck the endpoint at? I missed that - should be "/organizations/" or "/o/" to match the app, but that may be too late.

File: quantaq/endpoints/orgs/__init__.py [hunk 15]
```diff
@@ -0,0 +1,30 @@
+from quantaq.endpoints import Domain
+
+
+class Orgs(Domain):
```
Comment (dhhagan) [inline quantaq/endpoints/orgs/__init__.py]: Let's update to Organizations please

**Review by dhhagan — CHANGES_REQUESTED** [review]
The only major comment/request is to use the correct name for Organizations so customers aren't confused/aren't using "orgs" instead of "organizations"

**Comment by lswainemoore** [comment]
@dhhagan I believe I've updated them all if you want to take a look. I could also squash commits to get them out if you'd like, though I suppose this PR would document that...

**Review by dhhagan — APPROVED** [review]

**Comment by dhhagan** [comment]
> @dhhagan I believe I've updated them all if you want to take a look. I could also squash commits to get them out if you'd like, though I suppose this PR would document that...

Looks great. Good to merge and deploy!

**Pull request merged**